ci/verify-locked-down-signatures.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -eu
+shopt -s nullglob
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
For more strictness you could declare this constant as read-only:

```suggestion
readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Also suggested in your coding guidelines for Bash:
https://github.com/mullvad/coding-guidelines/issues/17